yarn.lock

```diff
@@ -5584,27 +5557,20 @@ axios-mock-adapter@1.19.0:
     fast-deep-equal "^3.1.3"
     is-buffer "^2.0.3"
 
-axios@0.19.2, axios@^0.19.0, axios@^0.19.2:
+axios@0.21.1:
+  version "0.21.1"
+  resolved "https://registry.yarnpkg.com/axios/-/axios-0.21.1.tgz#22563481962f4d6bde9a76d516ef0e5d3c09b2b8"
+  integrity sha512-dKQiRHxGD9PPRIUNIWvZhPTPpl1rf/OxTYKsqKUDjBwYylTvV7SjSHJb9ratfyzM6wCdLCOYLzs73qpg5c4iGA==
+  dependencies:
+    follow-redirects "^1.10.0"
+
+axios@^0.19.0, axios@^0.19.2:
```

Comment:
Do we know where these are still coming in from? What's the result of 'yarn why' for these package versions?

Reply:
The dependencies are `bundlewatch`, which we're using the latest version of, and `chromedriver`, which would be a major version bump so we'll leave this for now.